cogs/assistance-cmds/vccompatlists.wiiu.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Wii U inject Compatibility list sites
+help-desc: Lists various links to Wii U VC compatibility sites
+aliases: compatlists
+---
+
+You can see what games are compatibile with the Wii U/WiiVC from these sites.
```

Comment:
None of these have vWii VC.
```suggestion
You can see what games are compatibile with the Wii Us Virtual Console from these sites.
```

Reply:
Wii and GameCube is what I meant by "WiiVC"

Reply:
i think "Wii VC" implies vWii VC, especially with the seperation between "Wii U" and "Wii"

Reply:
Right